packages/cli/templates/platform-app-tailwind/src/index.tsx

```diff
@@ -1,13 +1,19 @@
-import { StrictMode } from 'react';
-import ReactDOM from 'react-dom/client';
 import { App } from './App';
+import { defineApp } from '@frontify/platform-app';
 import 'tailwindcss/tailwind.css';
 import '@frontify/fondue/style';
 import './main.css';
 
-// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-ReactDOM.createRoot(document.getElementById('root')!).render(
-    <StrictMode>
-        <App />
-    </StrictMode>,
-);
+export default defineApp({
+    app: App,
+    settings: {
```

Comment:
For the templates prefer adding a `settings.ts` which `export const settings = defineSettings({ ...`